config/rbac/auth_proxy_role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: proxy-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: controller-manager
```

Comment:
I don't think we use this role binding, but does the manager pod run under a different service account now?

Reply:
yes I think so